src/components/screen/Friend.tsx

```diff
@@ -63,7 +81,8 @@ export default function Screen(): ReactElement {
           alignSelf: 'stretch',
         }}
         contentContainerStyle={
-          friends.length === 0
+          // prettier-ignore
```

Comment:
Do we actually need this? Is this because of optional chaining?

Reply:
sorry I've missed that to review. @godon019 please explain why this is needed?

Reply:
there's nothing happened removing these code. I think this is legacy code so I removed it

Reply:
resolved [305d9f1](https://github.com/dooboolab/hackatalk-mobile/pull/115/commits/305d9f1c7740160adfa7351e175ef3415c883e96)